src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,87 @@
+import classNames from 'classnames';
+import React from 'react';
+
+interface Props {
+  setFilter: (filter: string) => void;
+  filter: string;
+  listOfActiveTodos: number;
+  hasActiveTodo: boolean;
+  deleteCompletedTasks: () => void;
+  inputRef: React.RefObject<HTMLInputElement>;
+}
+
+export const TodoFooter: React.FC<Props> = ({
+  setFilter,
+  filter,
+  listOfActiveTodos,
+  hasActiveTodo,
+  deleteCompletedTasks,
+  inputRef,
+}) => {
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {listOfActiveTodos} items left
+      </span>
+
+      {/* Active link should have the 'selected' class */}
+      <nav className="filter" data-cy="Filter">
+        <a
+          href="#/"
+          className={classNames('filter__link', {
+            selected: filter === 'All',
+          })}
+          data-cy="FilterLinkAll"
+          onClick={e => {
+            e.preventDefault();
+            setFilter('All');
+          }}
+        >
+          All
+        </a>
+
+        <a
+          href="#/active"
+          className={classNames('filter__link', {
+            selected: filter === 'Active',
+          })}
+          data-cy="FilterLinkActive"
+          onClick={e => {
+            e.preventDefault();
+            setFilter('Active');
+          }}
+        >
```

Comment:
Use `Object.values(your created enum)` and render these options with `map()` method